akka-actor/src/main/resources/reference.conf

```diff
@@ -411,7 +411,11 @@ akka {
         # work distribution approach is disabled and only the hash based is
         # used irrespective of the number of unique actors. Valid range is
         # 0 to 2048 (inclusive)
-        fair-work-distribution-threshold = 128
+        queue-selector = "akka.dispatch.affinity.FairDistributionHashCache"
+
+        # When using the "akka.dispatch.affinity.FairDistributionHashCache" queue selector
+        # the threshold is how many Runnables' indicies it will cache at most.
+        fair-work-distribution.threshold = 128
```

Comment:
l418: this parameter changed name (suffix threshold now takes a period rather than hyphen). Intended? (Less importantly now: the explaining comments (lines 411-413) are now further away from related parameter, and indicies (l 417) is conventionally spelled indices)